static/js/services/report-view-model-generator.js

```diff
@@ -0,0 +1,65 @@
+var _ = require('underscore');
+
+angular.module('inboxServices').factory('ReportViewModelGenerator',
+  function(
+    DB,
+    FormatDataRecord,
+    LineageModelGenerator
+  ) {
+    'ngInject';
+    'use strict';
+
+    var getFields = function(results, values, labelPrefix, depth) {
+      if (depth > 3) {
```

Comment:
This was copied from somewhere else. It just means we don't want to have nested fields go across the whole page in the UI.